corehq/apps/data_dictionary/util.py

```diff
@@ -416,3 +417,12 @@ def get_used_props_by_case_type(domain):
                 props_by_case_type[case_type_bucket.key] = []
             props_by_case_type[case_type_bucket.key].append(prop_bucket.key)
     return props_by_case_type
+
+
+def update_url_query_params(url, params):
+    """Adds query params to the url. Overrides the value if param already exists."""
+    parsed_url = urlparse(url)
+    current_params = dict(parse_qsl(parsed_url.query))
+    merged_params = urlencode({**current_params, **params})
+    # Note: _replace is a public method of namedtuple. Starts with _ to avoid conflicts with field names.
```

Comment:
Interesting! I found [the documentation on _replace()](https://docs.python.org/3/library/collections.html#collections.somenamedtuple._replace) (there are others too). I don't love it, but I appreciate the problem. `<cough>`case properties`</cough>` LOL